src/Aspire.Hosting/Dashboard/ConsoleLogPublisher.cs

```diff
@@ -76,4 +78,11 @@ LogsEnumerable SubscribeContainerResource(ContainerSnapshot container)
             return new DockerContainerLogSource(container.ContainerId);
         }
     }
+
+    private static async LogsEnumerable SubscribeGenericResource()
+    {
+        await Task.Yield();
```

Comment:
Is this needed? Remove or add a comment saying why

Reply:
Our analyzers, I guess I could suppress the warning.

Reply:
Leaving it with a comment is fine. I brought it up because I didn't know why it was needed